Update repositories.txt by adding Ligtht Task scheduler

belem2050 opened the pull request.

per1234 reviewed (commented):
Hello @belem2050. I am confirming your request.

Unlike submissions, this type of request is still handled manually. The Library Manager index maintainer occasionally comes through to clear all the outstanding requests. They will merge this pull request when it is done. You will get a notification from GitHub at that time.

After that, you can monitor the indexing of the library via its dedicated logs web page.
Information about that here:
https://github.com/arduino/library-registry/blob/main/FAQ.md#can-i-check-on-library-releases-being-added-to-library-manager

belem2050 commented:
@ArduinoBot

belem2050 commented:
> Hello @belem2050. I am confirming your request.
> 
> Unlike submissions, this type of request is still handled manually. The Library Manager index maintainer occasionally comes through to clear all the outstanding requests. They will merge this pull request when it is done. You will get a notification from GitHub at that time.
> 
> After that, you can monitor the indexing of the library via its dedicated logs web page. Information about that here: https://github.com/arduino/library-registry/blob/main/FAQ.md#can-i-check-on-library-releases-being-added-to-library-manager

@per1234 , thanks for your reactivity. It is true that I've changed the the repository name but I've also fixed typo that came into equation while committing. So I fixed it and wanted to deletethe first added lib in order to have it starting from version 1.0.0. That was why I asked for a lib removal. I don't know the under hood but I think if we just change the the url, it still won't work when it will be available in production.

per1234 commented:
> I've also fixed typo that came into equation while committing. So I fixed it and wanted to deletethe first added lib in order to have it starting from version 1.0.0. That was why I asked for a lib removal. I don't know the under hood but I think if we just change the the url, it still won't work when it will be available in production.

Just fix the bug, bump the version, then make a new release of the library. You can do that by following the instructions here:

https://github.com/arduino/library-registry/blob/main/FAQ.md#how-can-i-publish-a-new-release-once-my-library-is-in-the-list

MatteoPologruto reviewed (approved):
Done:
```
2025/05/12 15:24:23 Loaded 7992 libraries from DB
Changing URL of library LightTaskScheduler from https://github.com/belem2050/TaskScheduler.git to https://github.com/belem2050/LightTaskScheduler.git
Success!
```
